frontend/src/model.tsx

```diff
@@ -33,6 +34,8 @@ const axiosInstance = axios.create({
   baseURL: getHost(),
 })
 
+const logger = new PcmLogger(axiosInstance, getState(['app', 'appConfig']))
```

Comment:
issue: The logger would most likely be used throughout the app, also it depends on the value present in the store. By initializing it here, you'd never update the value of `appConfig` once it reached the store (it depends on an API call which is fired only after this line is executed)

I would suggest that we create a `LoggerContext` to pass the logger down to components. This would allow to react to changes in the store (e.g. the appConfig) and change the logger instance (if needed)

It's also easier to inject in component tests, as you'd simply wrap the component under test with a mocked `LoggerProvider`

In case we needed to reach out for the logger in places where contexts are not available (e.g. pure business logic), we can typically retrieve the logger from the context and pass it to such pure functions (or setup a service container to mimic an IoC container, but I'd wait for the actual need to occur)